dm/master/server.go

```diff
@@ -253,6 +267,21 @@ func errorCommonWorkerResponse(msg string, worker string) *pb.CommonWorkerRespon
 
 // StartTask implements MasterServer.StartTask
 func (s *Server) StartTask(ctx context.Context, req *pb.StartTaskRequest) (*pb.StartTaskResponse, error) {
+	log.L().Info("receive request and will save it to etcd", zap.Stringer("payload", req), zap.String("request", "StartTask"))
```

Comment:
can we remove `will save it to etcd`, because we must to save it into etcd

Reply:
removed f0931fb